<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Micmac selection tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icones/icones.qrc">
    <normaloff>:/MM/Icons/images/icon_32.png</normaloff>:/MM/Icons/images/icon_32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(200,200,200);
}

QMenu
{
	background: rgb(200,200,200);
}

QMenu::separator 
{
     height: 2px;
     background: rgb(100,100,100);
     margin-left: 10px;
     margin-right: 5px;
 }

QMenuBar 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(255,255,255), stop:1 rgb(200,200,200));
 }

QStatusBar
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(200,200,200), stop:1 rgb(255,255,255));
}
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QFrame" name="OpenglLayout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuStandard_views">
     <property name="title">
      <string>Standard views</string>
     </property>
     <addaction name="actionSetViewTop"/>
     <addaction name="actionSetViewFront"/>
     <addaction name="actionSetViewLeft"/>
     <addaction name="actionSetViewBack"/>
     <addaction name="actionSetViewRight"/>
     <addaction name="actionSetViewBottom"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuStandard_views"/>
    <addaction name="actionHelpShortcuts"/>
    <addaction name="actionFullScreen"/>
    <addaction name="actionShow_axis"/>
    <addaction name="actionShow_ball"/>
    <addaction name="actionShow_bounding_box"/>
    <addaction name="actionShow_cams"/>
    <addaction name="actionShow_help_messages"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Selection</string>
    </property>
    <addaction name="actionToggleMode_selection"/>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove_from_selection"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelect_none"/>
    <addaction name="actionReset"/>
    <addaction name="actionInvertSelected"/>
    <addaction name="separator"/>
    <addaction name="actionInsertPolylinepoint"/>
    <addaction name="actionDeletePolylinepoint"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_plys"/>
    <addaction name="actionLoad_camera"/>
    <addaction name="actionLoad_and_Export"/>
    <addaction name="actionLoad_image"/>
    <addaction name="actionSave_selection"/>
    <addaction name="actionSave_masks"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose_all"/>
    <addaction name="actionExit"/>
    <addaction name="actionRecentFiles"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuSelection"/>
  </widget>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionHelpShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
   <property name="toolTip">
    <string>Shortcuts</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewFront">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewYpos.png</normaloff>:/images/ccViewYpos.png</iconset>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewBack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewYneg.png</normaloff>:/images/ccViewYneg.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewXpos.png</normaloff>:/images/ccViewXpos.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewXneg.png</normaloff>:/images/ccViewXneg.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewZpos.png</normaloff>:/images/ccViewZpos.png</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewBottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewZneg.png</normaloff>:/images/ccViewZneg.png</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionToggleMode_selection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Points selection</string>
   </property>
   <property name="toolTip">
    <string>Toggle between Points selection mode and standard mode</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="toolTip">
    <string>select all vertex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLoad_camera">
   <property name="text">
    <string>Open cameras</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionLoad_and_Export">
   <property name="text">
    <string>Load and Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShow_ball">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show ball</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionShow_axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show axis</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionShow_help_messages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show help messages</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShow_cams">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show cams</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add points/pixels</string>
   </property>
   <property name="toolTip">
    <string>Add points/pixels</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad_plys">
   <property name="text">
    <string>Open ply files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDeletePolylinepoint">
   <property name="text">
    <string>Delete Polyline point</string>
   </property>
   <property name="toolTip">
    <string>Delete Polyline point</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </action>
  <action name="actionSave_selection">
   <property name="text">
    <string>Save selection infos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelect_none">
   <property name="text">
    <string>Select none</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionInvertSelected">
   <property name="text">
    <string>Invert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRemove_from_selection">
   <property name="text">
    <string>Remove from selection</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionShow_bounding_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show bounding box</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRecentFiles">
   <property name="text">
    <string>Recent files:</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLoad_image">
   <property name="text">
    <string>Open image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_masks">
   <property name="text">
    <string>Save masks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLoad_image_mask">
   <property name="text">
    <string>Open image + mask</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionInsertPolylinepoint">
   <property name="text">
    <string>Insert Polyline point</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
